CMakeLists.txt

```diff
@@ -50,21 +50,21 @@ DEAL_II_INITIALIZE_CACHED_VARIABLES()
 
 OPTION(ENABLE_COUPLED_PROBLEMS "Enable coupled problems with 1d code." OFF)
 
-IF(${ENABLE_COUPLED_PROBLEMS})
-  MESSAGE("-- Looking for 1dsolver library")
-  SET(CMAKE_FIND_LIBRARY_PREFIXES "lib")
-  SET(CMAKE_FIND_LIBRARY_SUFFIXES ".so" ".a" ".lib" ".dll" ".dylib")
-  FIND_LIBRARY(FVM_LIB NAMES 1dsolver HINTS ${CMAKE_CURRENT_SOURCE_DIR})
-  IF(NOT FVM_LIB OR ${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
-    MESSAGE("-- Could not find 1dsolver library -- disabling coupled problems")
-    SET(ENABLE_COUPLED_PROBLEMS OFF)
-  ELSE()
-    MESSAGE("-- Found 1dsolver library -- enabling coupled problems")
-    MESSAGE("-- FVM_LIB: ${FVM_LIB}")
-  ENDIF()
-ELSE()
-  MESSAGE("-- Coupled problems disabled")
-ENDIF()
+# IF(${ENABLE_COUPLED_PROBLEMS})
+#   MESSAGE("-- Looking for 1dsolver library")
+#   SET(CMAKE_FIND_LIBRARY_PREFIXES "lib")
+#   SET(CMAKE_FIND_LIBRARY_SUFFIXES ".so" ".a" ".lib" ".dll" ".dylib")
+#   FIND_LIBRARY(FVM_LIB NAMES 1dsolver HINTS ${CMAKE_CURRENT_SOURCE_DIR})
+#   IF(NOT FVM_LIB OR ${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
+#     MESSAGE("-- Could not find 1dsolver library -- disabling coupled problems")
+#     SET(ENABLE_COUPLED_PROBLEMS OFF)
+#   ELSE()
+#     MESSAGE("-- Found 1dsolver library -- enabling coupled problems")
+#     MESSAGE("-- FVM_LIB: ${FVM_LIB}")
+#   ENDIF()
+# ELSE()
+#   MESSAGE("-- Coupled problems disabled")
+# ENDIF()
```

Comment:
Why did you comment these out?

Reply:
I commented it out locally, sorry. Fixed.